qiskit_optimization/algorithms/multistart_optimizer.py

```diff
@@ -52,6 +52,10 @@ def __init__(self, trials: int = 1, clip: float = 100.) -> None:
                 with the ``-clip`` or ``clip`` values correspondingly for the initial guesses.
         """
         super().__init__()
+        
+        if trials <= 0:
+            raise ValueError('Number of trials should be positive')
```

Comment:
How about
```suggestion
            raise ValueError(f'Number of trials should be 1 or higher, but was {trials}')
```
(Or whatever text you prefer such as a minimum of 1 etc) and it can be helpful to see the value that was actually passed, that caused the failure in the msg. Same for below.

I imagine CI will complain since it will want Raises:  added to the docstrings too

Reply:
So what do you suggest then?

Reply:
It indeed failed you look at the build details failure, plus there is some whitespace at end of line that is checked for - these are the errors
```
qiskit_optimization/algorithms/multistart_optimizer.py:118:89: C0303: Trailing whitespace (trailing-whitespace)
qiskit_optimization/algorithms/multistart_optimizer.py:41:4: W9006: "ValueError" not documented as being raised (missing-raises-doc)
qiskit_optimization/algorithms/multistart_optimizer.py:111:4: W9006: "ValueError" not documented as being raised (missing-raises-doc)
```
As I mentioned above you need to add Raises: to the docstrings of the the methods, here is an example for elsewhere in the optimization code https://github.com/Qiskit/qiskit-optimization/blob/3e21815034c72030eab17f2865895dc0d12076a4/qiskit_optimization/problems/quadratic_program.py#L289-L305